.changeset/odd-flies-rush.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/utils': major
+'@twilio-paste/core': minor
```

Comment:
It is indeed exported through core, good callout! Ordinarily this would be a major to core, however I asked Nora to mark it as a minor for two reasons:

- The first and most important is that this is undocumented & internal tooling. Anything that isn't documented is Use At Your Own Risk. It's kind of like pre-v1.0.0 in semver, which means breaking changes don't need to bump the major version.
- Second no one is using this according to our reporting scans. So no need to worry folks for nothing.